### DISCUSSION TITLE
Tooltip for upvote action

### DISCUSSION THREAD

**Discussion by mbiesiad** [community/community#31344] [Discussions]
Hi, it's really nice to have an 'upvote action' (as an arrow up with counter) here. I think it would be great to add a tooltips like "**upvote**" and "downvote / **undo**" for these relevant actions.

Here, if you click on 'emoji' icon nothing will happen until you select the emoji, but with click "arrow up" it will. Tooltips with tiny description is missing.

![image](https://user-images.githubusercontent.com/18367606/187415476-855ae6ee-15be-4d3a-bb03-7d48d36d52eb.png)

Best wishes,

**Comment by hariprasd** [comment]
You can post this as an Issue [here.](https://github.com/community/community/discussions/categories/issues) and Developers will give life to it

**Comment by mbiesiad** [comment]
Thanks for your comment here, your link redirect to: https://github.com/community/community/discussions/categories/issues
and I posted that here: https://github.com/community/community/discussions

It's an issue or feature request, but connected with `community/discussions`.

**Reply by hariprasd** [reply]
yes, when it is labelled as Issue, it will get noticed by Developers & It will be resolved faster.
